firebase-vscode/package.json

```diff
@@ -3,7 +3,7 @@
   "displayName": "firebase-vscode",
   "publisher": "firebase",
   "description": "VSCode Extension for Firebase",
-  "version": "0.0.9",
+  "version": "0.0.22",
```

Comment:
FYI I have to keep bumping the version every time I want to push to Monospace to test

Reply:
If monospace will allow it, consider using prerelease tags (ie `0.0.22-alpha.0`) instead of patch versions when testing

Reply:
Sounds good, I will give it a try.